src/plugins/interactive_setup/server/config.ts

```diff
@@ -13,4 +13,5 @@ export type ConfigType = TypeOf<typeof ConfigSchema>;
 
 export const ConfigSchema = schema.object({
   enabled: schema.boolean({ defaultValue: false }),
+  forceSetup: schema.boolean({ defaultValue: false }),
```

Comment:
__note:__ the necessity of this flag was discussed over Slack - for testing purposes, at least, we want to force interactive setup mode.